modules/oe_whitelabel_paragraphs/tests/src/Kernel/Paragraphs/ParagraphsTestBase.php

```diff
@@ -80,16 +85,18 @@ protected function setUp(): void {
    *
    * @param \Drupal\paragraphs\ParagraphInterface $paragraph
    *   Paragraph entity.
+   * @param string $langcode
+   *   Rendering language code, defaults to 'en'.
    *
    * @return string
    *   Rendered output.
    *
    * @throws \Exception
    */
-  protected function renderParagraph(ParagraphInterface $paragraph): string {
+  protected function renderParagraph(ParagraphInterface $paragraph, string $langcode = 'en'): string {
```

Comment:
```suggestion
  protected function renderParagraph(ParagraphInterface $paragraph, string $langcode = NULL): string {
```
This would match the view() method default parameter.